### ISSUE TITLE
hourly P

### ISSUE THREAD

**Issue by kbrannan** [dsobota/Upper_Yaquina_PEST#10]
is the hourly inorg and org P input constant and do you want to keep it constant in the q2k input shown below
INORGP_UP
https://github.com/dsobota/Upper_Yaquina_PEST/blob/853ed13909196380528a773e14fe33c4e33c46d4/python/UY_do.q2k#L283
ORGP_UP
https://github.com/dsobota/Upper_Yaquina_PEST/blob/853ed13909196380528a773e14fe33c4e33c46d4/python/UY_do.q2k#L284

**Comment by dsobota** [comment]
Yes, it is constant in the calibrated model.  I don't think there is enough diel variation to justify varying it in the q2k input.
